libc/common.c

```diff
@@ -60,3 +60,6 @@ int _read_file(const char *path, char *buf, size_t bufsz)
ravida ultricies. Etiam hendrerit justo ut venenatis hendrerit. Etiam vulputate erat vel ligula faucibus, quis posuere purus efficitur. Phasellus aliquet eros id nunc vulputate pharetra. Maecenas non euismod diam. Duis at sagittis tortor, id aliquam eros. Ut sed ligula velit. Donec vel magna ullamcorper, hendrerit elit vel, porttitor dui. Sed sapien tortor, vehicula euismod erat bibendum, dignissim viverra nibh. Vivamus et ex arcu. Name pharetra risus cursus mi interdum ullamcorper. Proin vel urna scelerisque est convallis lacinia. Mauris libero enim, consequat quis justo id, placerat ultricies nisi. Aliquam posuere tortor ut ipsum commodo, vitae efficitur eros ultrices. Aliquam gravida sodales quam, at sodales leo elementum eget. Fusce fermentum metus in bibendum euismod. Morbi porta imperdiet justo a posuere. Curabitur id risus risus. Praesent mollis blandit orci, eget semper ligula consectetur at. Phasellus vestibulum enim et diam dapibus, at suscipit libero pretium. Nulla facilisi. Fusce gravida dapibus tincidunt. Sed non dapibus eros, non efficitur nunc. Phasellus augue quam, rhoncus a sem a, blandit mollis nunc. Donec pharetra, orci quis euismod porttitor, orci nibh vulputate felis, eu dignissim eros mauris id dolor. Nullam dictum venenatis leo sed accumsan. Pellentesque various diam felis, sed feugiat nisi hendrerit volutpat. Ut faucibus efficitur sollicitudin. Nulla efficitur massa dignissim sem dignissim, vel dignissim sem elementum. Nullam semper a metus ac molestie. Aenean tempus risus lectus, eu semper ipsum lobortis sit amet. Etiam vel semper enim, vel lobortis augue. Quisque vehicula quam consectetur leo porttitor, quis cursus dolor posuere. Vestibulum semper ut orci vitae venenatis. Cras commodo mauris eu ligula auctor sagittis. Sed bibendum orci vitae interdum elementum. Praesent vehicula orci quis lectus accumsan ultrices. Proin vehicula mollis ipsum nec tristique. Nulla non lacus nec lacus luctus ullamcorper. Fusce libero quam, molestie non nisi eget, fermentum vestibulum urna. In ante ligula, tempor in neque id, faucibus tincidunt tortor. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Nullam suscipit rhoncus consequat. Maecenas fermentum molestie est non finibus. Cras auctor purus ac tellus euismod commodo. In faucibus dapibus hendrerit. Nulla dapibus quis ligula eget efficitur. Donec tempor quis est a fermentum. Maecenas ut dui elementum, lobortis nibh non, euismod nunc. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nunc ex nisl, vestibulum quis cursus sed, maximus vitae est. Quisque lobortis tellus metus, ut rhoncus ex dignissim vitae. Pellentesque inhabitant morbi tristique senectus et netus et malesuada fames ac turpis egestas. Suspendisse enim nisl, egestas nec posuere a, mattis ac nisi. Duis lobortis blandit bibendum. Donec ac lorem sed ante fringilla. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Lorem ipsum dolor sit amet, consectetur adipiscing elit. ";
```

Comment:
Please:
 - put it in separate file with only test data and include only if needed in tests
 -  add sensible prefix to all global test data (`hugeTestStr` is not specific enough, easy to confuse with local/static variables)
 - split the string into multiple lines to make it more readable (don't exceed ~120 column width)
 - imho for arrays always define companion const with array len (in this use case `const size_t hugeTestStrLen = sizeof(hugeTestStrLen);` or something similar)